src/test/kotlin/com/hibob/academy/unitests/keren/ListManagerTest.kt

```diff
@@ -0,0 +1,140 @@
+package com.hibob.academy.unitests.keren
+
+import org.junit.jupiter.api.Assertions.*
+import org.junit.jupiter.api.Test
+
+class ListManagerTest{
+    @Test
+    fun `Test adding a unique person`(){
+        assertEquals(true,ListManager().addPerson(Person("Ron Azar", 24)))
+    }
+
+    @Test
+    fun `Test adding a duplicate person and ensure it throws the expected exception`(){
+        assertThrows(IllegalArgumentException::class.java) {
```

Comment:
You should better move these 2 first lines before the assert
            val managers = ListManager()
            managers.addPerson(Person("Ron Azar", 24))

This way you'll verify where the exception was thrown from